src/global/munge_engines.R

```diff
@@ -415,7 +415,7 @@ munge_time_component <-  function(network, domain, site_name, prodname_ms, track
         #_only_ run it here. that will take some investigation though. for one
         #thing, we'd need to consider giant bind_rows operations above when
         #operating in high-res mode
-        out_filt <- synchronize_timestep(out_filt)
+        # out_filt <- synchronize_timestep(out_filt)
```

Comment:
@spencerrhea We need this for the situations described in the comments above. Was it causing trouble?

Reply:
Ok I didn't realize synchronize_timestep() was in both munge_combined() and munge_time_component(). It was causing issues for plum, the only domain where munge_time_component() was being used (now it is used in catalin_jemez too). This is because carry_uncertainty() was not being run in the munge kernel and synchronize_timestep() will have an error if there is no error associated with the val  column. It was not being run in the munge kernel because the chemistry data was split between products and I wanted synchronize_timestep() to run with all chemistry data in the data frame, not a subset. So both carry_uncertainty and synchronize_timestep() are being run in the derive kernel where all the chemistry data is combined for plum.

So the issue described in the comment is being addressed in the derive kernels. Does that make sense?

For the second part of the comment '_only_ run it here.'. I think that makes sense to only run synchronize_timestep() in the munge engine as opposed to in the munge kernels. I can make that change, would have to change a good chunk of kernels but wouldn't take too long and would make things a little cleaner and less redundant. Thoughts?